src/components/Header/Header.styled.jsx

```diff
@@ -0,0 +1,289 @@
+    color: #858598;
+    @media (max-width: 531px){
+        border-right: 1px solid white;
+        color: white;
+    }
+`;
+
+// ----------- styled for - HeaderMobile ----------
+export const MobileHamburgerLine = styled.div`
+    border-bottom: 1px solid #858598;
+`;
+
+export const MobileHamburgerContainer = styled.nav`
+    position: relative;
+    width: 22px;
+    height: 14px;
+    border-top: 1px solid #858598;
+    display: flex;
+    flex-direction: column;
+    border-bottom: 1px solid #858598;
+    justify-content: center;
+    cursor: pointer;
+    &:hover {
+        width: 23px;
+        height: 15px;
+        border-top: 1px solid black;
+        border-bottom: 1px solid black;
+        box-shadow:         
+        rgb(0 0 0 / 19%) 1px 3px 20px 0px, 
+        inset -10px -10px 15px rgb(255 255 255 / 80%), 
+        inset 10px 10px 15px rgb(70 70 70 / 12%);
+        ${MobileHamburgerLine}{
+            border-bottom: 1px solid black;
+        }
+    }
+    @media (max-width: 300px){
+        margin-left:20px;
+    }
+`;
+
+// -------------------------------------------------
+export const MobileHamburgerbackground = styled.nav`
+    position: fixed;
+    width: 274px;
+    height: 100%;
+    right: 0px;
+    top: 0px;
+    background: #FFBC33;
+    box-shadow: -15px 0px 20px rgba(0, 0, 0, 0.25);
+    Z-index: 10;
+    overflow: hidden;
+    transition: 0.5s;
+`;
+
+export const MobileHamburgerSectionStyle = css`
+    height: 44px;
+    width: 100%;
+    border-bottom: 1px solid #ffffff;
+    display: grid;
+    align-items: center;
+    padding-left: 20px;
+    ${fontRegular}
+    &:hover {
+    box-shadow: 
+    rgb(0 0 0 / 19%) 1px 3px 10px 0px, 
+    inset -6px -6px 8px rgb(255 255 255 / 80%), 
+    inset 6px 6px 8px rgb(70 70 70 / 12%);
+  }
+`;
+
+
+export const MobileHamburgerSectionMenuUp = styled.div`
+${MobileHamburgerSectionStyle}
+    margin-top: 80px;
+    border-top: 1px solid #ffffff;
+`;
+
+export const MobileHamburgerSectionMenu = styled.div`
+${MobileHamburgerSectionStyle}
+`;
+
+export const MobileHamburgerLoginBox = styled.div`
+    position: absolute;
+    margin-top: 18px;
+`;
+
+export const MobileHamburgerCross = styled.div`
+    height: 22px;
+    width: 1px;
+    background-color: #ffffff;
+    transform: rotate(45deg);
+    Z-index: 11;
+    position: absolute;
+`;
+export const MobileHamburgerCross1 = styled.div`
+    height: 22px;
+    width: 1px;
+    background-color: #ffffff;
+    transform: rotate(90deg);
+    Z-index: 12;
+    position: absolute;
+`;
+
+export const MobileHamburgerCrossBox = styled.div`
+    height: 22px;
+    width: 22px;
+    top: 20px;
+    right: 20px;
+    padding-left: 11px;
+    position: absolute;
+    &:hover {
+        box-shadow: 
+        inset rgb(0 0 0 / 19%) 0px 0px 10px 0px,
+        rgb(0 0 0 / 19%) 1px 3px 10px 0px;
+        border-radius: 50px;
+        padding-left: 10.5px;
+        padding-top: 1.2px;
+        cursor: pointer;
```

Comment:
this custom cross animation is cool, but I think we should strictly adhere to our mockup

Reply:
I will remove this hover)